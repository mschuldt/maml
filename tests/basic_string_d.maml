arduino = Arduino(desktop=True)

@block(once)
def test():
    x = "test string"
    print_s(x)
    die(1)
    
arduino.send(test)
