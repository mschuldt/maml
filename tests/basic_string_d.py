arduino = Arduino(desktop=True)

@block
def test():
    x = "test string"
    print_s(x)
    die(1)
    
arduino.send(test)
