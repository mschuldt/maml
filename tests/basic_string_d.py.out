Initializing avm...
Ready.

test string
bye.
